[Docs] translate 6_benchmarks.md into Chinese

soonera opened the pull request:
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Translate 6_benchmarks.md into Chinese.

## Modification

Translate 6_benchmarks.md into Chinese.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

soonera committed:
[Docs] translate 6_benchmarks.md into Chinese

fangyixiao18 commented on docs/zh_cn/tutorials/6_benchmarks.md:
```diff
@@ -1,185 +1,186 @@
-# Tutorial 6: Run Benchmarks
+# 教程 6：运行基准
```
Better to change `基准` to `基准评测`, check the whole file

soonera committed:
Update 6_benchmarks.md

change 基准 to 基准评测

Jiahao000 commented on docs/zh_cn/tutorials/6_benchmarks.md:
```diff
@@ -1,185 +1,186 @@
+## 分类
 
-As for classification, we provide scripts in folder `tools/benchmarks/classification/`, which has 4 `.sh` files, 1 folder for VOC SVM related classification task and 1 folder for ImageNet nearest-neighbor classification task.
+关于分类，我们在`tools/benchmarks/classification/`文件夹中提供了脚本，其中有4个 `.sh` 文件和1个文件夹用于 VOC SVM 相关的分类任务。
```
The translation of `1 folder for ImageNet nearest-neighbor classification task` is missing here.

Jiahao000 commented on docs/zh_cn/tutorials/6_benchmarks.md:
```diff
@@ -1,185 +1,186 @@
 ### VOC SVM / Low-shot SVM
 
-To run these benchmarks, you should first prepare your VOC datasets. Please refer to [prepare_data.md](https://github.com/open-mmlab/mmselfsup/blob/master/docs/en/prepare_data.md) for the details of data preparation.
+为了运行这个基准评测，你应该首先准备你的 VOC 数据集，数据预准备的细节请参考[prepare_data.md](https://github.com/open-mmlab/mmselfsup/blob/master/docs/en/prepare_data.md)。
```
`数据预准备` -> `数据准备`

soonera committed:
Update 6_benchmarks.md

(1)  Add Chinese translation of  ‘1 folder for ImageNet nearest-neighbor classification task’
(2) 数据预准备 -> 数据准备
